Formula/ghostscript.rb

```diff
@@ -59,6 +55,11 @@ class Ghostscript < Formula
     sha256 "0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401"
   end
 
+  patch do
+    url "http://git.ghostscript.com/?p=ghostpdl.git;a=patch;h=830afae5454dea3bff903869d82022306890a96c"
+    sha256 "b0b7a042e50f3eb5f7de8f8e833397953f8730030c71572ae372f44f7144c9bb"
+  end
```

Comment:
What was this patch for? It no longer produces the same checksum:
```
❯ curl -sL http://git.ghostscript.com/\?p\=ghostpdl.git\;a\=patch\;h\=830afae5454dea3bff903869d82022306890a96c | sha256sum
9c282e3f732b5c182ab761e187850d9f4c29d938a93e3785d9cb6ced13111d6c  -
```

Reply:
I do not think I actually added this myself -- I believe @SMillerDev added this?